spec/features/feedback_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+RSpec.feature 'Customer feedback', js: true do
+  around do |example|
```

Comment:
Will inline.. but just so you know.. `run` is an alias of `call` :) https://github.com/rspec/rspec-core/blob/master/lib/rspec/core/example.rb#L340